CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -443,10 +443,10 @@ CFTypeRef _CFRuntimeCreateInstance(CFAllocatorRef allocator, CFTypeID typeID, CF
     memset(&memory->_cfinfoa, 0, size - (sizeof(memory->_cfisa) + sizeof(memory->_swift_rc)));
 
     // Set up the cfinfo struct
-    uint32_t *cfinfop = (uint32_t *)&(memory->_cfinfoa);
+    uint64_t *cfinfop = (uint64_t *)&(memory->_cfinfoa);
```

Comment:
The type of `_cfinfoa` is different on different targets.  IIRC, its if its android or 64-bit, its 64-bit, otherwise 32-bits.

Reply:
It is different for non-Swift runtimes, but it always 64 for the Swift runtime (and this code only applies to the Swift runtime). See https://github.com/apple/swift-corelibs-foundation/blob/master/CoreFoundation/Base.subproj/CFRuntime.h#L198